docs/README.md

```diff
@@ -235,7 +235,7 @@ Display the version number.
 
 ### IO
 
-#### `Fly.prototype.source (...globs)`
+#### `Fly.prototype.source (...globs, options)`
```

Comment:
I know it is, but I coded around it to make this work the same way the jQuery.extend method works where you can pass in anything you want and if the last argument is a plain object then it is removed from the `globs`. If the last argument is a `string` or an `array` then `options` becomes an empty `{}`